app/lib/efile/line_data.yml

```
@@ -490,6 +490,16 @@ IT214_LINE_33:
   label: '33 Enter the amount from line 32 or 31, whichever is less. This is the credit for your household.'
 IT227_PART_2_LINE_1:
   label: ''
+MD502_LINE_1:
+  label: "Adjusted gross income from your federal return"
```

Could you add the line number before the line description like the other labels, for example:
```suggestion
  label: "1 Adjusted gross income from your federal return"
```